src/Sulu/Component/SmartContent/Orm/BaseDataProvider.php

```diff
@@ -126,16 +126,30 @@ public function resolveResourceItems(
 
     /**
      * Resolves filters.
+     *
+     * @param null $limit
+     * @param int $page
+     * @param null $pageSize
+     * @param array $options
+     *
+     * @return array
      */
     private function resolveFilters(
         array $filters,
-        $locale,
+        string $locale,
```

Comment:
aslong as the called interface `$this->repository->findByFilters(` does not have a typehint I would not add a typehint here. and set it as phpdoc. else I'm afraid that something will break

Reply:
```suggestion
        $locale,
```